new_client/src/components/auth/Signin.jsx

```diff
@@ -1,12 +1,20 @@
-import React from 'react';
-import { Container, Paper, Grid, TextField, Button } from '@material-ui/core';
+import React, {useState, useEffect} from 'react';
+import { Container, Paper, Grid, TextField, Button, Typography, Link } from '@material-ui/core';
```

Comment:
Again, personal opinion, so not so sure, but shouldn't we stick to 1 UI library? Somewhere we are using MaterialUI, and someplaces we are using Bootstrap. @PrabhuPratik27 What do you feel?